marketplace/deployer_helm_tiller_base/marketplace-k8s-app-tools-helm-hooks/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Would suggest renaming the chart to simply `application` and moving this to this directory:

```
/marketplace/charts/application
```

This directory essentially is the source for https://cloud-marketplace-tools.storage.googleapis.com/charts

Reply:
Moved chart definitions to `marketplace/charts` and renamed to `marketplace-hooks`.